src/NuGet.Clients/PackageManagement.VisualStudio/ProjectSystems/LegacyCSProjPackageReferenceProject.cs

```diff
@@ -168,8 +180,8 @@ private IScriptExecutor ScriptExecutor
                 _project.AddOrUpdateLegacyCSProjPackage(
                     packageId,
                     range.MinVersion.ToNormalizedString(),
```

Comment:
is this really a real scenario? how could we install a package with minVerion as null in range?